concrete-core/src/specification/engines/glwe_ciphertext_conversion.rs

```diff
@@ -0,0 +1,35 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::GlweCiphertextEntity;
+
+engine_error! {
+    GlweCiphertextConversionError for GlweCiphertextConversionEngine @
+}
+
+/// A trait for engines converting glwe ciphertexts.
+///
+/// # Semantics
+///
+/// This [pure](super#operation-semantics) operation generates a glwe ciphertext containing the
+/// conversion of the `input` glwe ciphertext to a different representation.
+///
+/// # Formal Definition
+pub trait GlweCiphertextConversionEngine<Input, Output>: AbstractEngine
+where
+    Input: GlweCiphertextEntity,
+    Output: GlweCiphertextEntity<KeyFlavor = Input::KeyFlavor>,
```

Comment:
Yes ! 

I tried to come with a different word for that because I already used `Kind` for `EntityKind` and I preferred not to mix words for very different things. What do you think about it ?

Reply:
see https://github.com/zama-ai/concrete_internal/issues/177

Reply:
What about `KeyDistribution` ?